src/libraries/System.Collections.Concurrent/src/System/Collections/Concurrent/BlockingCollection.cs

```diff
@@ -1763,10 +1763,7 @@ private static void ValidateMillisecondsTimeout(int millisecondsTimeout)
         /// <exception cref="System.ObjectDisposedException">If the collection has been disposed.</exception>
         private void CheckDisposed()
```

Comment:
Note that I deleted such helpers when they were only called from a couple of places, but kept them when they were called from many.